FirebaseRemoteConfig/Sources/FIRRemoteConfig.m

```diff
@@ -329,11 +331,14 @@ - (void)activateWithCompletion:(FIRRemoteConfigActivateChangeCompletion)completi
     // New config has been activated at this point
     FIRLogDebug(kFIRLoggerRemoteConfig, @"I-RCN000069", @"Config activated.");
     [strongSelf->_configContent activatePersonalization];
-    // Update activeRolloutMetadata
-    [strongSelf->_configContent activateRolloutMetadata];
     // Update last active template version number in setting and userDefaults.
+    [strongSelf->_settings updateLastActiveTemplateVersion];
+    // Update activeRolloutMetadata
+    NSArray<NSDictionary *> *activeRolloutMetadata =
+        [strongSelf->_configContent activateRolloutMetadata];
     dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0), ^{
```

Comment:
I am not sure about the async/sync here. 
When Crashlytics register with FirebaseSession, there is a sync call to send the first notification. Should we also follow this pattern?
https://github.com/firebase/firebase-ios-sdk/blob/main/FirebaseSessions/Sources/FirebaseSessions.swift#L253